tests/testthat/test-renderer2-showSelected-same.R

```diff
@@ -25,8 +25,10 @@ test_that("redundant aes not saved to tsv", {
   geom.tsv <- Sys.glob(
     file.path("animint-htmltest", "geom1_point_points_*.tsv"))
   expect_equal(length(geom.tsv), 1)
-  expected.names <- sort(c("x", "y", ss.vec))
-  tsv.data <- read.table(geom.tsv, header=TRUE)
+  expected.names <- sort(c("x", "y", ss.vec,
+                           "alpha", "colour", "fill",
+                           "shape", "size", "stroke"))
```

Comment:
@tdhock Thanks for bringing this up! It was one of the matters I wanted to discuss. I'll see how I can implement this with the new ggplot2 internals.